.github/workflows/coverage.yml

```diff
@@ -0,0 +1,128 @@
+# Coverage build
+name: Coverage
+
+on: workflow_call
+
+permissions:
+  contents: read
+
+env:
+  BUILD_DIR : "${{github.workspace}}/build"
+  INSTL_DIR : "${{github.workspace}}/../install-dir"
+  COVERAGE_DIR : "${{github.workspace}}/coverage"
+
+jobs:
+  Coverage:
+    name: Coverage build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          fetch-depth: 0
+
+      - name: Install dependencies (ubuntu-latest)
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y lcov
+
+      - name: Download file exports-coverage-basic-ubuntu-20.04-shared-OFF-hwloc-OFF
+        uses: actions/download-artifact@v4
```

Comment:
use specific hash commit for the version like with upload

Reply:
Done.